src/main/java/dk/alexandra/fresco/lib/helper/builder/AbstractProtocolBuilder.java

```diff
@@ -67,6 +71,10 @@
 	 */
 	@Override
 	public void beginParScope() {
```

Comment:
Also, it would make the code easier to read. I really think we should do it that way.